pandas/core/generic.py

```diff
@@ -38,6 +38,7 @@
 import pandas.core.algorithms as algos
 from pandas.core.base import PandasObject, SelectionMixin
 import pandas.core.common as com
+from pandas.core.computation.common import _remove_spaces_column_name
```

Comment:
import this locally in the function (as we have some restricted import about computation)

Reply:
Done